packages/strapi/lib/core-api/service.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
```

Comment:
Modifying this file isn't necessaey. same as modifying the content manager services. This code is agnostic to the primaryKey. You can either send id or idAttribute key and it will be handled the same way by the query

Reply:
Ah yes, but we still need it for entry object. Resolved this.